### DISCUSSION TITLE
Screen Saver Mode?

### DISCUSSION THREAD

**Discussion by snydoodlsh** [djphazer/O_C-Phazerville#34] [Ideas]
I have 1.6.6 o_C-phazerville-v1.6.6+stock1+VOR-b708bbe1.hex file installed on my Plum Audio OC. It doesn't seem like the screensaver is working? I've become more sensitive to the screensaver mode, after noticing my other LCD screen modules showing burn in... anyways. The screensaver that appears is a full screen of 4 column notes... it would be nice if it was just blank or had some cool random pattern that kept any unique app layout from burning in. I've seen videos here and there that show some cool swirling stochastic pattern around a center / gravity point. That would be dope as the default screen saver for Phaserville. Cheers.

**Comment by djphazer** [comment]
My apologies - I shipped the new Hemisphere screensaver view in v1.6.4, but I realize now there's still demand for a blank screen. You make a good point about the solid lines potentially being burned in... I found it kinda harsh to look at as well, so I pushed a tweak that changes them to softer dotted lines (in case you're keen on building from source!)

When I find time, I'm going to implement either a secondary screen-blanking mode or a simple toggle to disable the screensaver in Hemisphere. I may also explore a less static view like you suggested, something more swirly, maybe a starfield ;)

**Reply by snydoodlsh** [reply]
Yeah - something that isn't static would be dope. I've come across certain simulations and old videos of older firmware... I think they are visualizations of chaos attractors? Gotta make that O&C look good without any burn in! 

<img width="1272" alt="Screenshot 2023-11-21 at 2 52 55 PM" src="https://github.com/djphazer/O_C-BenisphereSuite/assets/142067228/756a7dfc-4ff2-47ed-b1bb-107974d471dd">

**Comment by djphazer** [comment]
How about something like this?

![InShot_20231124_134031522.gif](https://github.com/djphazer/O_C-BenisphereSuite/assets/109086194/ed64a85c-6b0a-48b2-a5fc-5efd3b9024c4)

I also added a config option so it can be turned off (blank screen):

![PXL_20231124_181829613~2.jpg](https://github.com/djphazer/O_C-BenisphereSuite/assets/109086194/cc1e4849-fe5a-4609-892d-5530a74e0dc8)

**Comment by snydoodlsh** [comment]
Wow - that's awesome! Zaps!

**Comment by snydoodlsh** [comment]
Just updated to the latest firmware. The screensaver zaps is awesome! Thanks!

**Reply by djphazer** [reply]
oh yeah, are you on T3.2? Zaps is still in the code, but temporarily displaced by the Scope view. I'll bring it back, I promise ⚡

**Reply by snydoodlsh** [reply]
OH - it's all good. I just upgraded to 1.8. You got this cool star field thing. That's good enough! No need to bring back zaps just for me. As long as there is a cool screensaver, I'm happy!
